app/lib/LevelBus.coffee

```diff
@@ -240,6 +241,8 @@ module.exports = class LevelBus extends Bus
     # don't let peeking admins mess with the session accidentally
     return unless @session.get('creator') is me.id
     return if @session.fake
+    if @changedSessionProperties.code
```

Comment:
Should we move this check to `updateSessionConcepts()` too?  Similar to the codeLanguage check.

Reply:
I wanted `updateSessionConcepts` to definitely run if it can. The difference between this check and the codeLanguage check is the latter is a showstopper.